.gitpod.yml

```diff
@@ -0,0 +1,21 @@
+---
+image:
+  file: container/gitpod/Dockerfile
+
+tasks:
+  - name: Lint the Javascript dependencies
+    before: npm i
+    command: npm run lint
+  - name: Build openQA
+    before: |
+      bash -x tools/ci/build_cache.sh
+      bash -x tools/ci/build_autoinst.sh
+      sudo chown -R gitpod $(pwd)
+    command: tools/ci/run_unit_tests.sh fullstack
```

Comment:
openQA does not need to be "built". And what you do is run the fullstack test, but only this one? So what is this for?

Reply:
Mostly to pull in all the dependencies and build os-autoinst, so that the user can start right away testing out their changes.

Reply:
that answers the first part and the last question. But what you do is run the fullstack test, but only this one?